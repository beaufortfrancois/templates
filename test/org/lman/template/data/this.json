{
  "list": [1, 3, 2],
  "string": "hello"
}
