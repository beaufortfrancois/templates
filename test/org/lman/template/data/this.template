Here are the numbers in order:{{#list}} {{string}} {{@}}{{/}}
