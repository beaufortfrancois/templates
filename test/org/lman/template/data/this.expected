Here are the numbers in order: hello 1 hello 3 hello 2
